main.py

```diff
@@ -18,12 +18,28 @@
     TokenUsage,
     LanguageModels,
     ErrorResponse,
+    EmbeddingRequest,
+    EmbeddingResponse,
 )
+
+
+# todo: make this importable from top level
+# from llmspec.llmspec import EmbeddingData
+# temporary fix: embedding attr type
+class EmbeddingData(msgspec.Struct):
+    embedding: List[float] | List[List[float]]
+    index: int
+    object: str = "embedding"
```

Comment:
You can use the latest llmspec code from your branch. Just mark this PR as blocked by another one.